CompatBot/Commands/Misc.cs

```diff
@@ -25,6 +25,8 @@ internal sealed class Misc: BaseCommandModuleCustom
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed",
+            "Much yes", "Make it so", "Consider it done", "Totally", "You bet",
+            "Sure", "Yup", "Yep", "Okie dokie", "Certainly", "Yarp",
```

Comment:
```suggestion
            "Yup", "Yep", "Certainly", "Yarp",
```
"okie dokie" is not really a positive answer, but an acknowledgement on the bot's part, so it doesn't really fit
also "yup" is already present, but in a slightly different form